src/scenes/home/opCodeCon/opCodeCon.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import commonUrl from 'shared/constants/commonLinks';
+import LinkButton from 'shared/components/linkButton/linkButton';
+import styles from './opCodeCon.css';
+
+const OpCodeCon = () => (
+  <div className={styles.hero}>
+    <div className={styles.heading}>
+      <h1>
+        OpCodeCon
+      </h1>
+      <h3>
+        Join us for our innaugural Operaction Code Convention!
+      </h3>
+      <p>September 19th-20th, 2018</p>
+      <p>Raleigh Convention Center, Raleigh, NC</p>
+      <p>
+        <a
+          href="mailto:staff@operationcode.org"
+          target="_blank"
+          rel="noopener noreferrer"
+          className={styles.textLink}
+        >
```

Comment:
No need for `target="_blank"` or `rel="noopener noreferrer"` here, this is a mailto link for an email address that should open in a user's default email application when clicked. Those attributes are typically used for external hypertext links to web pages that you want to open in a separate tab.

Reply:
Just for my information, I don't have a default email enabled so when I removed both of these it doesn't do anything. Is that the right behavior?